dev-aws/kafka-shared/iam_credentials.tf

```diff
@@ -21,3 +21,41 @@ module "iam_credentials_producer" {
 
   cert_common_name = "auth-customer/credentials-api"
 }
+
+module "iam_credentials_consumer" {
+  source = "../../modules/consumer"
+
+  topic          = kafka_topic.iam_credential_v1.name
+  consumer_group = "indexer-iam-credentials-v1"
+
+  cert_common_name = "auth-customer/iam-credentials-v1-indexer"
+}
+
+resource "kafka_acl" "indexer_iam_credentials_v1_topic_access" {
```

Comment:
By using the consumer module, we don't really need these individual resources, as they're included in the module

Reply:
https://github.com/utilitywarehouse/kafka-cluster-config/pull/72/commits/4d6fb6630ae2f8ab6b2e0f9fa3653d69c8460b34